src/hotspot/share/runtime/thread.cpp

```diff
@@ -3668,12 +3681,17 @@ void Threads::initialize_java_lang_classes(JavaThread* main_thread, TRAPS) {
   if (CallTestMethod) {
```

Comment:
Good as is. 
NB. CallTestMethod could be a string, specifying the class/method combination.